src/channel/playlist.js

```diff
@@ -694,7 +694,7 @@ PlaylistModule.prototype.handleJumpTo = function (user, data) {
         this.channel.logger.log("[playlist] " + user.getName() + " skipped " + title);
 
         if (old && old.temp && old !== to) {
-            this._delete(old.uid);
+            this._delete(old.uid, user.getName());
```

Comment:
Similar, I think it is weird to say that someone who skips from a temporary video to another one "deleted" the temporary video (this also applies below in the case where the server advances to the next item in the list which triggers the removal of the temporary item).